atomsci/ddm/pipeline/parameter_parser.py

```diff
@@ -1750,7 +1745,7 @@ def make_dataset_key_absolute(parsed_args):
     # if so, make it relative to current working directory
     # update to allow for datastore
     if not parsed_args.datastore:
-        if (not parsed_args.dataset_key is None) and (not os.path.isabs(parsed_args.dataset_key)):
```

Comment:
`not parsed_args.dataset_key` was probably always `True`. But also it would never be `None`